src/github/SyftGithubAction.ts

```diff
@@ -43,9 +43,18 @@ export function getArtifactName(): string {
   const format = getSbomFormat();
   let extension: string = format;
   switch (format) {
```

Comment:
suggestion, but not blocking: should we update these to match all syft options? That is, we only advertise `cyclonedx` but silently allow for `cyclonedx-xml` as well?